.github/workflows/autodocs-platform.yaml

```diff
@@ -108,31 +105,33 @@ jobs:
       id: octo-sts
       with:
         scope: chainguard-dev/edu
-        identity: edu
+        identity: autodocs
```

Comment:
seems this sts policy https://github.com/chainguard-dev/edu/blob/main/.github/chainguard/autodocs.sts.yaml need to update to match this workflow